server/chat-plugins/calculator.ts

```diff
@@ -132,17 +139,26 @@ export const commands: ChatCommands = {
 	math: "calculate",
 	calculate(target, room, user) {
 		if (!target) return this.parse('/help calculate');
+		const targets = target.split(',');
+		const expression = targets[0];
+		let base = parseInt(targets[1]);
+		if (isNaN(base)) base = 10;
+		if (targets.length > 1 && !(base in BASE_PREFIXES)) {
```

Comment:
```suggestion
		if (!(base in BASE_PREFIXES)) {
```

`base` should be `10` if `targets.length <= 1`